### PR TITLE
Added solution for Scrambled-words challenge

### PR DESCRIPTION
**Pull request by gteti**
I've created this repo to add my proposed solution to the Reply training week1 Scrambled words challenge.

### CODE REVIEW

**Comment by gteti** [comment]
Anyone please check if it is good to merge in 4 days, Thanks

**Comment by gmacario** [comment]
Sorry but I cannot merge this PR unless it passes some automated tests such as <https://github.com/github/super-linter>

**Comment by gteti** [comment]
#2 has been merged. Let's wait for 4 days before merging and in the meantime let's fix eventually problems.

File: training/week1/coding/scrambled-words/README.md
Comment (gmacario) [inline training/week1/coding/scrambled-words/README.md:24]: I would suggest to enclose the flag into a "verbatim" section, such as

```text
{FLG:70c42b6d6818f3141865f10d1608f7a9763138554f6d5120662222189854231c}
```

so that the flag can be easily copied and then pasted into the CSC online platform

**Review by gmacario — CHANGES_REQUESTED** [review]

File: training/week1/coding/scrambled-words/README.md
Comment (gmacario) [inline training/week1/coding/scrambled-words/README.md:2]: Please add the following metadata

* Category: Coding
* Points: 100

**Review by gmacario — CHANGES_REQUESTED** [review]

**Review by gmacario — APPROVED** [review]

**Pull request merged**
